lib/contentProfileListener.js

```diff
@@ -5,7 +5,7 @@ const {SHRContentProfileLexer} = require('./parsers/SHRContentProfileLexer');
 const {SHRContentProfileParser} = require('./parsers/SHRContentProfileParser');
 const {SHRContentProfileParserListener} = require('./parsers/SHRContentProfileParserListener');
 const {SHRErrorListener} = require('./errorListener.js');
-const {Specifications, Version, ContentProfile, ContentProfileRule, Identifier, IdentifiableValue, ChoiceValue, PrimitiveIdentifier, PRIMITIVES} = require('shr-models');
+const {Specifications, Version, NamespaceContentProfile, ContentProfile, ContentProfileRule, Identifier, IdentifiableValue, ChoiceValue, PrimitiveIdentifier, PRIMITIVES} = require('shr-models');
```

Comment:
It doesn't seem like `NamespaceContentProfile` is an actual model, and it's only in this file here once. Am I missing something? Or perhaps this is a relic of a previous attempt at working on this task?

Reply:
You are not missing something, it is a relic of a previous attempt. Fixed in https://github.com/standardhealth/shr-text-import/pull/106/commits/024592ed27de536c10cf9eca60055dfd609173ba.